omi_api/views/compositions.py

```diff
@@ -39,6 +39,8 @@ def post(self):
                             location='json')
         parser.add_argument('publishers', type=list, required=True,
                             location='json')
+        parser.add_argument('iswc', type=str, required=True,
```

Comment:
I'll make `required=False`. However, then matching recordings and compositions won't work for this implementation.

@vrde and I talked.